### PR TITLE
#49: TruncatedText

### PR DESCRIPTION
**Pull request by fabriciofx**
As per #49:

- `TruncatedText` introduced.

### CODE REVIEW

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/text/SubText.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    public SubText(final Text text, final Scalar<Integer> start,
+        final Scalar<Integer> end) {
+        this.origin = text;
+        this.start = start;
+        this.end = end;
+    }
+
+    @Override
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
+    public String asString() throws IOException {
```
Comment (yegor256) [inline src/main/java/org/cactoos/text/SubText.java]: @fabriciofx why not just throw `Exception`?

File: src/main/java/org/cactoos/text/TruncatedText.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+/**
+ * Truncate a Text to a max width.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Fabricio Cabral (fabriciofx@gmail.com)
+ * @version $Id$
+ * @since 0.6
+ */
+public final class TruncatedText implements Text {
```
Comment (yegor256) [inline src/main/java/org/cactoos/text/TruncatedText.java]: @fabriciofx what is the point of this class? Why `SubText` is not enough?

**Comment by yegor256** [comment]
@fabriciofx good idea, see some comments above

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/text/TruncatedText.java (hunk 2 above)
Comment (fabriciofx) [inline src/main/java/org/cactoos/text/TruncatedText.java]: @yegor256 Makes sense. I'll delete `TruncatedText` in favor of `SubText`. Thanks for your attention!

**Comment by fabriciofx** [comment]
Closing this PR in favor of SubText (PR #194).

**Pull request closed without merging**